File: ncarg2d/src/libncarg_gks/awi/gsfasi.f
C
C	$Id: gsfasi.f,v 1.5 2008-07-27 00:21:02 haley Exp $
C                                                                      
C                Copyright (C)  2000
C        University Corporation for Atmospheric Research
C                All Rights Reserved
C
C The use of this Software is governed by a License Agreement.
C
      SUBROUTINE GSFASI (STYLI)
C
C  SET FILL AREA STYLE INDEX
C
      INTEGER ESFASI
      PARAMETER (ESFASI=37)
C
      include 'gkscom.h'
C
      INTEGER STYLI
C
C  Check if GKS is in the proper state.
C
      CALL GZCKST(8,ESFASI,IER)
      IF (IER .NE. 0) RETURN
C
C  Check that the index is valid.
C
      IF (STYLI .LE. 0) THEN
        ERS = 1
        CALL GERHND(84,ESFASI,ERF)
        ERS = 0
        RETURN
      ENDIF
C
C  Set the current index in the GKS state list.
C
      CFASI = STYLI
C
C  Invoke the workstation interface.
C
      FCODE = 39
      CONT  = 0
      CALL GZROI(0)
      IL1   = 1
      IL2   = 1
      ID(1) = STYLI
      CALL GZTOWK
      IF (RERR .NE. 0) THEN
        ERS = 1
        CALL GERHND(RERR,ESFASI,ERF)
        ERS = 0
      ENDIF
C
      RETURN
      END

File: ncarg2d/man/gks/gsfasi.m
.\"
.\"	$Id: gsfasi.m,v 1.16 2008-12-23 00:03:03 haley Exp $
.\"
.TH GSFASI 3NCARG "March 1993" UNIX "NCAR GRAPHICS"
.SH NAME
GSFASI (Set fill are style index) - sets the fill area style index.
.SH SYNOPSIS
CALL GSFASI (STYLI)
.SH C-BINDING SYNOPSIS
#include <ncarg/gks.h>
.sp
void gset_fill_style_ind(Gint fill_style_ind);
.SH DESCRIPTION
.IP STYLI 12
(Integer, Input) - 
Specifies workstation-dependent options for hatch or pattern fills. 
Pattern fill options are not supported in NCAR Graphics. Hatch fill 
options are:
.RS
.IP 1 
Horizontal lines (default)
.IP 2 
Vertical lines
.IP 3 
Lines of positive slope
.IP 4 
Lines of negative slope
.IP 5 
Horizontal and vertical lines
.IP 6 
Lines of positive and negative 
slope
.RS
.SH ACCESS
To use GKS routines, load the NCAR GKS-0A library 
ncarg_gks.
.SH SEE ALSO
Online: 
gfa, gsfais, gscr, gsfaci, gqfais, gqfasi, 
areas, gset_fill_style_ind
.sp
Hardcopy: 
"User's Guide for NCAR GKS-0A Graphics"
.SH COPYRIGHT
Copyright (C) 1987-2009
.br
University Corporation for Atmospheric Research
.br

The use of this Software is governed by a License Agreement.
